src/main/java/com/algorand/sdkutils/RunAlgodV2Tests.java

```diff
@@ -0,0 +1,42 @@
+package com.algorand.sdkutils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileReader;
+
+import com.algorand.algosdk.v2.client.common.Client;
+import com.algorand.sdkutils.generators.TestGenerator;
+import com.algorand.sdkutils.generators.Utils;
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class RunAlgodV2Tests extends TestGenerator {
```

Comment:
To include these tests as part of the integration test suite, move them to the test directory and use the integration test classname format (i.e. `RunAlgodV2TestsIT`).